modules/huawei/include/hw_ivs.h

```diff
@@ -36,9 +39,16 @@ class HuaweiIVS : public VMSInterface {
                           const std::string &nvr_code,
                           const std::string &transport) override;
 
+  void sdk_playback(const std::string &camera_code,
+                    const std::string &start_time,
+                    const std::string &end_time);
+
  private:
   int _session_id;
   bool _logged_in;
+
+  IVS_ULONG _playback_handle = 0;
+  HWND _show_box;
```

Comment:
HWND is also Windows specific. Insert these line in the top, after includes.

```c++
#ifndef WIN32
#define HWND void*
#endif
```

Reply:
Is it a typo? `WIN32` or `_WIN32`?

Reply:
Use `_WIN32`